recipes/trisbm/yum_requirements.txt

```diff
@@ -0,0 +1,10 @@
+libX11
```

Comment:
This file probably isn't needed

Reply:
right

Reply:
I see you added it again, was this intentional?

Reply:
it is no longer there in the latest commit